### PR TITLE
Add BNPL sites

### PR DESCRIPTION
**Pull request by fletcherdodd**
Adding BNPL (Buy Now Pay Later) sites; LatitudePay, Humm & Laybuy

### CODE REVIEW

File: entries/l/latitudepay.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "LatitudePay": {
+    "domain": "latitudepay.com",
+    "contact": {
+      "facebook": "latitudefinancialservices",
```
Comment (phallobst) [inline entries/l/latitudepay.com.json]:
```suggestion
      "facebook": "LatitudePay",
```

File: entries/l/latitudepay.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      "twitter": "latitude_fs"
+    },
+    "keywords": [
+      "payments"
+    ]
```
Comment (phallobst) [inline entries/l/latitudepay.com.json:14]:
```suggestion
    ],
    "regions": [
      "au"
    ]
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/l/latitudepay.com.json (hunk 1 above)
Comment (fletcherdodd) [reply entries/l/latitudepay.com.json]: I would argue that the original one is the better option as it has more followers and is actually the verified account/promoted by the company

Comment (phallobst) [reply entries/l/latitudepay.com.json]: On their [main page](https://www.latitudepay.com/), the linked Facebook account is "LatitudePay" - it doesn't get more official than that, I suppopse.

Comment (fletcherdodd) [reply entries/l/latitudepay.com.json]: Fair, you win this round

**Review by phallobst — APPROVED** [review]

**Pull request merged**